events/janus_wsevh.c

```diff
@@ -713,17 +709,21 @@ static int janus_wsevh_callback(struct lws *wsi, enum lws_callback_reasons reaso
 			ws_client->buflen = 0;
 			ws_client->bufpending = 0;
 			ws_client->bufoffset = 0;
-			reconnect_retries = 0;
+			reconnect_delay = 0;
+			reconnect = FALSE;
 			janus_mutex_init(&ws_client->mutex);
 			lws_callback_on_writable(wsi);
+			JANUS_LOG(LOG_INFO, "WebSocketsEventHandler connected\n");
 			return 0;
 		}
 		case LWS_CALLBACK_CLIENT_CONNECTION_ERROR: {
 			JANUS_LOG(LOG_ERR, "Error connecting to backend (%s)\n",
 				in ? (char *)in : "unknown error");
-			/* Should we reconnect? */
+			janus_wsevh_calculate_reconnect_delay_on_fail();
 			disconnected = janus_get_monotonic_time();
 			reconnect = TRUE;
+			JANUS_LOG(LOG_WARN, "WebSocketsEventHandler: Error attempting connection... (next retry in %ds)\n", reconnect_delay);
```

Comment:
We already have a log error above: please add the retry info there.

Reply:
ack